### ISSUE TITLE
I meet some problems when i run 'python main.py'!!!!!!!!!!

### ISSUE THREAD

**Issue by meihuameii** [smallcowbaby/OmniAnomaly#32]
![image](https://user-images.githubusercontent.com/48877543/110436714-82818f00-80ef-11eb-9b26-0d8ba09a0075.png)
The problem happened in wrapper.py  '__init__()' function

**Comment by musaceylan** [comment]
+1

**Comment by Anxia10** [comment]
These are parameters that need to be set by yourself

**Comment by ilwoof** [comment]
> These are parameters that need to be set by yourself

Could you explain how I can set these 6 parameters? For example, for the SMD dataset. Thanks a lot.

**Comment by ServetKocak** [comment]
@ilwoof are you able to solve the issue?

**Comment by ilwoof** [comment]
> @ilwoof are you able to solve the issue?

yes, please refer to https://github.com/astha-chem/mvts-ano-eval. @ServetKocak

**Comment by ServetKocak** [comment]
Hi @ilwoof , thank you for getting me back. It helped. Nut right now I am having other issues while trying to get the lstm cell output since rnn.static_rnn is removed from the new version of tensorflow. How did you deal with it?

**Comment by GPla** [comment]
There is a `requirement.txt` in the repo. Why aren't installing the given version of `tensorflow=1.12.0`?
